src/frontend/src/lib/canisters/backend.errors.ts

```diff
@@ -0,0 +1,27 @@
+import type {
+	BtcAddPendingTransactionError,
+	SelectedUtxosFeeError
+} from '$declarations/backend/backend.did';
+import { CanisterInternalError } from '$lib/canisters/errors';
+
+export const btcPendingTransactionError = (response: BtcAddPendingTransactionError) => {
```

Comment:
Note: in some cases, it makes sense to have one function per several related endpoints, e.g. - `btcAddPendingTransaction` and `btcGetPendingTransaction` share a common error type on the BE side, therefore 2 error parsers are not needed.